doc/tutorial/argument_parser/disable_version_check.cpp

```diff
@@ -2,6 +2,6 @@
 
 int main(int argc, char ** argv)
 {
-    seqan3::argument_parser myparser{"Game-of-Parsing", argc, argv, false};
-    // disable version checks permanently ----------------------------^
+    seqan3::argument_parser myparser{"Game-of-Parsing", argc, argv, seqan3::update_notifications::disabled};
+    // disable update notifications permanently ----------------------------^
```

Comment:
So permanentely means here that setting the update notification off is encoded in the binary and therefore never asks on the command line? Is that clear from the tutorial that this happens?

Reply:
> So permanentely means here that setting the update notification off is encoded in the binary and therefore never asks on the command line?

yes

> Is that clear from the tutorial that this happens?

In the tutorial I refer to the documentation of `seqan3::argument_parser` were I have an extra section 
![image](https://user-images.githubusercontent.com/7955878/96128920-38a4cc00-0ef6-11eb-866f-be99c6e73a4d.png)

So I think it is clear?